### PR TITLE
docs: Prevent console access

### PR DESCRIPTION
**Pull request by hifabienne**
Documentation about how to prevent users from accessing ZITADEL Management Console.

### Definition of Ready

- [ ] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: Prevent users from accessing ZITADEL console
+sidebar_label: Disable ZITADEL Console
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:3]:
```suggestion
sidebar_label: Limit access to console
```

Comment (mffap) [reply docs/docs/guides/solution-scenarios/disable-console.mdx:3]: We're not really disabling console

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+---
+
+ZITADEL does include the ZITADEL Console, which allows you to manage all ZITADEL resources as well as end users to manage their profile information.
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:6]:
```suggestion
ZITADEL includes a console that allows Managers to configure all resources. All uses, including end-users, by default, view and manage their profile information.
```

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+In some use cases you want to prevent users from accessing the console, this could be:
+- User management is integrated into your own app
+- Users login via SSO and should not be able to change their data in the Console
+- Only managers should be able to access Console to manage their users
+
+At the moment it is not possible to simply disable the Console but with a workaround you can prevent users from accessing the Console.
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:12]:
```suggestion
At the moment it is not possible to simply disable the Console but with the configuration described in this guide you can prevent users from accessing the Console.
```

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## Self-hosted
+
+If you are running ZITADEL self-hosted we recommend to restrict the access to ZITADEL Console via WAF/reverse-proxy for Non-Manager users
+
+## ZITADEL Cloud (and self-hosted)
+
+### Default redirect url
+
+The Default Redirect URI will be used, if a user calls the login page directly. More specifically, typically a client will initiate login with an auth request.
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:22]:
```suggestion
The Default Redirect URI will be used, if a user calls the login page directly. More specifically, typically a client will initiate login with an [auth request](https://zitadel.com/docs/apis/openidoauth/authrequest).
```

Comment (mffap) [reply docs/docs/guides/solution-scenarios/disable-console.mdx:22]: Should be actually linked and explained in https://github.com/zitadel/zitadel/issues/7399

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+If there is no auth request, users will be redirected to the Default Redirect URI, which is by default https://<custom_domain>/ui/console/
+
+
+Change default redirect url of instance: https://<custom_domain>/ui/console/settings?id=login
+
+![Login Policy Advanced Setting: Default Redirect URL](/img/guides/solution-scenarios/console-default-redirect.png)
+
+### Restricting Console in default-project
+
+With this workaround, you will limit users from accessing the default-project, if they are not explicitly granted to use this project.
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:33]: Can we explain default-project? Or even better add that as "Concept" for reference?

File: docs/docs/guides/solution-scenarios/disable-console.mdx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+  "name": "ZITADEL",
+  "projectRoleAssertion": false,
+  "projectRoleCheck": false,
+  "hasProjectCheck": true,
+  "privateLabelingSetting": "PRIVATE_LABELING_SETTING_UNSPECIFIED"
+}'
+```
+
+Where
+- CUSTOM_DOMAIN is your domain, f.e. b2b-demo-rbxajm.zitadel.cloud
```
Comment (mffap) [inline docs/docs/guides/solution-scenarios/disable-console.mdx:77]:
```suggestion
- CUSTOM_DOMAIN is your domain, f.e. mydemo.zitadel.cloud
```

**Review by mffap — CHANGES_REQUESTED** [review]
Thanks, that will be helpful!

I would for sure explain the Default Redirect URL on the settings page and link it there. That is more consistent and the documentation is anyways missing/needed.

To avoid confusion, we should also explain default-project. I think it might be best to do so as Concept, since we can reference it across the docs.

**Comment by mffap** [comment]
@hifabienne we should maybe link this guide in the overview for console and mention that end user access can be restricted by following the configuration in this guide. 

https://zitadel.com/docs/guides/manage/console/overview

**Comment by mffap** [comment]
> > @hifabienne we should maybe link this guide in the overview for console and mention that end user access can be restricted by following the configuration in this guide.
> > https://zitadel.com/docs/guides/manage/console/overview
> 
> I have added the same guide also in the menu in the console: image
> 
> But can also mention it in the overview there

If you can, I would avoid the link in the sidebar (make the user jump to somewhere else...) and rather provide the information in the text.

File: docs/docs/guides/solution-scenarios/disable-console.mdx (hunk 5 above)
Comment (hifabienne) [reply docs/docs/guides/solution-scenarios/disable-console.mdx]: added

**Review by mffap — APPROVED** [review]

**Pull request merged**